QUARKUS-6258.md

```diff
@@ -40,12 +47,16 @@ According to Mario, this is intentional, since they consider upstream lang4chain
 Ollama modules can be started on the laptop in dev mode. Jlama ones can be run as well.
 The features we want to cover are being used in integration-tests modules for OpenAI (Chats, Chat Memory, user/system messages, Function calling/Tools, Guardrails), RAG,. There is no coverage for WebSockets (only in codestarts) and for ChatModel (only in mocktests).
 
-### Impact on test suite
+## Impact on test suite
 We will focus on updating and running upstream tests from the Quarkiverse extensions against productised binaries.
 
 Primary focus will be on RHEL-based testing to ensure functionality of proposed features against productized bits. For the OpenShift side, we will explore the possibility of using the `@GitRepositoryQuarkusApplication` approach.
 
-As we aim for tech preview level for 3.27, we won't be focusing on full platform matrix coverage. We will be gaining experience to expand the coverage and platforms in the upcoming releases.d testing.
+As we aim for tech preview level for 3.27, we won't be focusing on full platform matrix coverage. We will be gaining experience to expand the coverage and platforms in the upcoming releases.
+
+## Impact on test suites for Full Support
+Since Quarkus AI may be moved to full support in its first release (Quarkus 3.27), we need to have a full support on all platforms. This means, that all tests should be in Quarkus Test suite, to guarantee running an all supported platforms.
+Additionally, we should add verification for this extension to Marete and startstop tests.
 
 ## Impact on resources
 The tests themselves do not consume much resources (single ci.m1.medium/ci.m1.large should be enough).
```

Comment:
We usually also have time estimation of test runtime.

This will also increase for full support, so let's have the accounting updated, and broken down per test suite.

Reply:
Let's start with an accounting of what jobs are supposed to be changed/added, time impact can be added once you have an idea